whotracksme/data/assets/trackerdb.sql

```diff
@@ -3287,6 +3430,81 @@ INSERT INTO "trackers" VALUES('userzoom.com','UserZoom',3,'https://www.userzoom.
+INSERT INTO "trackers" VALUES('contentstream.pl','Content Stream',8,'http://contentstream.pl/','netsprint',NULL,NULL);
+INSERT INTO "trackers" VALUES('dailymail.co.uk','Daily Mail',8,'http://www.dailymail.co.uk/home/index.html','dmg_media',NULL,NULL);
+INSERT INTO "trackers" VALUES('sociaplus.com','Insider',6,'http://useinsider.com/','insider','2868',NULL);
+INSERT INTO "trackers" VALUES('salesmanago.pl','SALESmanago',4,'https://www.salesmanago.com/','sales_manago','2403',NULL);
+INSERT INTO "trackers" VALUES('zebestof.com','Zebestof',4,'http://www.zebestof.com/en/home/','zebestof',NULL,NULL);
+INSERT INTO "trackers" VALUES('wysistat.com','Wysistat',6,'http://wysistat.net/','wysistat','902',NULL);
+INSERT INTO "trackers" VALUES('o2.pl','o2.pl',8,'https://www.o2.pl/','o2.pl',NULL,NULL);
+INSERT INTO "trackers" VALUES('omniconvert.com','Omniconvert',4,'https://www.omniconvert.com/','omniconvert',NULL,NULL);
+INSERT INTO "trackers" VALUES('optinproject.com','OptinProject',4,'https://www.optincollect.com/en','optincollect',NULL,NULL);
+INSERT INTO "trackers" VALUES('ora.tv','Ora.TV',4,'http://www.ora.tv/','ora.tv','2846',NULL);
+INSERT INTO "trackers" VALUES('owox.com','OWOX',6,'https://www.owox.com/','owox_inc',NULL,'Google Analytics partner');
+INSERT INTO "trackers" VALUES('peer5.com','Peer5',9,'https://www.peer5.com/','peer5',NULL,NULL);
+INSERT INTO "trackers" VALUES('pepper.com','Pepper',4,'https://www.pepper.com/','6minutes',NULL,NULL);
+INSERT INTO "trackers" VALUES('pladform.ru','Pladform',4,'https://distribution.pladform.ru/','pladform',NULL,NULL);
+INSERT INTO "trackers" VALUES('playbuzz.com','Playbuzz',6,'https://www.playbuzz.com/','playbuzz','2469',NULL);
+INSERT INTO "trackers" VALUES('powr.io','POWr',6,'https://www.powr.io/','powr',NULL,NULL);
+INSERT INTO "trackers" VALUES('prismamediadigital.com','Prisma Media Digital',4,'http://www.pmdrecrute.com/','prisma_media_digital','1223',NULL);
+INSERT INTO "trackers" VALUES('privy.com','Privy',6,'https://privy.com/','privy',NULL,NULL);
+INSERT INTO "trackers" VALUES('pscp.tv','Periscope',7,'https://www.pscp.tv/','periscope',NULL,NULL);
+INSERT INTO "trackers" VALUES('purch.com','Purch',6,'http://www.purch.com/','purch','1165',NULL);
+INSERT INTO "trackers" VALUES('push.world','Push.world',2,'https://push.world/en','push.world',NULL,NULL);
+INSERT INTO "trackers" VALUES('qq.com','qq.com',8,'http://www.qq.com/','qq.com',NULL,NULL);
+INSERT INTO "trackers" VALUES('quartic.pl','Quartic',6,'https://www.quarticon.com/','quarticon','2132',NULL);
+INSERT INTO "trackers" VALUES('rcs.it','RCS',8,'http://www.rcsmediagroup.it/','rcs','1535',NULL);
+INSERT INTO "trackers" VALUES('recettes.net','Recettes.net',8,'http://www.recettes.net/','recettes.net',NULL,NULL);
+INSERT INTO "trackers" VALUES('redcdn.pl','Red CDN',9,'http://www.atendesoftware.pl/','atende_software',NULL,NULL);
```

Comment:
This seems to be called redGalaxy CDN from their product page: http://www.atendesoftware.pl/multimedia/technology/redcdn

Reply:
Updated.